codegen.yml

```diff
@@ -0,0 +1,8 @@
+schema: data/schema.graphql
```

Comment:
Let's add a TODO to move this to some shared location to keep in sync with MatterhornApiService.

Reply:
added todo